src/api_client.js

```diff
@@ -12,6 +12,7 @@ const API_ENDPOINTS = {
   production: 'https://submarine.discolabs.com/api/v1',
   staging: 'https://submarine-staging.discolabs.com/api/v1',
   uat: 'https://submarine-uat.discolabs.com/api/v1',
+  dev: 'https://submarine-aparna.au.ngrok.io/api/v1'
```

Comment:
@appu4ever i don't think we need to add this.   If there are handy steps which might help a developer run this in a development environment, feel free to add those steps to the README.

Reply:
@tomcdisco, Gavin has mentioned that he would like the README to not include any disco developer specific information. But I agree, it would help developers if it is clearly specified what information needs to go where exactly. I have included a comment in the client.js file for this purpose for now. Happy to change if that doesn't suffice.